internal/handlers/handlers_test.go

```diff
@@ -1,15 +1,29 @@
 package handlers
 
 import (
+	"bytes"
+	"encoding/json"
 	"github.com/stretchr/testify/assert"
 	"io"
 	"net/http"
 	"net/http/httptest"
+	"shortener/config"
```

Comment:
Please look at the following link https://go.dev/doc/code
Please sort out imports